ansible-inventory-origin-15-aio

```diff
@@ -0,0 +1,24 @@
+
+[OSEv3:children]
+nodes
+masters
+
+[OSEv3:vars]
+ansible_connection=local
+openshift_release=v1.5
+openshift_hosted_logging_deploy=true
+openshift_logging_install_logging=true
+short_version=1.5
+openshift_deployment_type=origin
+openshift_logging_namespace=logging
+openshift_logging_image_prefix=docker.io/openshift/origin-
+openshift_master_identity_providers=[{'challenge': 'true', 'login': 'true', 'kind': 'AllowAllPasswordIdentityProvider', 'name': 'allow_all'}]
+openshift_logging_es_cluster_size=1
+openshift_logging_image_version=v1.5.0-rc.0
+deployment_type=origin
+
+[nodes]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
+
+[masters]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
```

Comment:
This inventory file is for a local only, all-in-one deployment.  For the first step, I wanted to keep it as simple as possible, and it has proven difficult to set up a remote machine properly in order to use ansible to install/configure it remotely.  Not only that, but openshift-ansible requires a very specfic version of ansible (at least, until recently there were some bugs).

If you know ansible, you can fill out the [masters] and [nodes] sections with real, remote server names and IP addresses. (and remove the ansible_connection=local).

More instructions will be forthcoming, but for now, baby steps.